documentation/src/main/asciidoc/topics/con_near_caches.adoc

```diff
@@ -0,0 +1,35 @@
+[id='near-caches-{context}']
+= Near Caches
+Near caches are local to Hot Rod clients and store recently used data so that every read operation does not need to traverse the network, which significantly increases performance.
+
+Near caches are populated on calls to `get()` or `getVersioned()`.
+When entries are updated or removed in remote caches on {brandname} Servers, those entries are invalidated in the near cache.
+If entries are requested after they are invalidated, clients must retrieve them from the remote caches again.
+
+Near caches are also cleared when clients fail over to different servers.
+
+.Bounded near caches
+
+You should always use bounded near caches by specifying the maximum number of entries they can contain.
+When near caches reach the maximum number of entries, eviction automatically takes place to remove older entries.
+This means you do not need to manually keep the cache size within the boundaries of the client JVM.
+
+[IMPORTANT]
+====
+Do not use maximum idle expiration with near caches because near-cache reads do not propagate the last access time for entries.
+====
+
+.Bloom filters
+
+Bloom filters optimize performance for write operations by reducing the total number of invalidation messages.
+
+Bloom filters use a client listener that is registered on the {brandname} Server to keep the near cache in sync.
+For this reason, bloom filters require the connection pool configuration to:
```

Comment:
The reason the connection pool config is required is because we currently have to keep track of which connection maps to which client. I don't know if we want to mention this, but this restriction may be removed in future versions if a client identifies itself upon connecting to the server.

Reply:
Thanks, Will. I think it might be better if we don't go too into detail with that. It's probably better to just say what the requirements are and not open a can of worms.